src/webchat-ui/components/plugins/message/regular/index.tsx

```diff
@@ -2,15 +2,25 @@ import * as React from 'react';
 import { MessagePlugin, MessageComponentProps } from '../../../../../common/interfaces/message-plugin';
 import MessageBubble from '../../../presentational/MessageBubble';
 
-const regularMessageBot = 'regular-message bot';
-const regularMessageUser = 'regular-message user';
+const className = (source: string) => {
```

Comment:
we should rename this to `messageClassName` and put this into the `RegularMessage` component (switch to a curly-brace-with-return-syntax). There is a [classname](https://www.npmjs.com/package/classname) package which does something else and i was irritated.

Reply:
fixed